[Bug]: weekly CI, test_chat_completions_invalid_requests[qwen3_omni-invalid_response_format_json_schema], AssertionError: Expected error text to contain all of ('response_format', 'value_error', 'json_schema'); missing ['value_error']

### Your current environment

<details>
<summary>The output of <code>python collect_env.py</code></summary>

```text
CI env
```

</details>


### Your code version

<details>
<summary>The commit id or version of vllm</summary>

```text
0.27.0
```
</details>
<details>
<summary>The commit id or version of vllm-omni</summary>

```text
9798bc3a8d5a400b97fba7402f42510577e7302d
```
</details>


### 🐛 Describe the bug

https://buildkite.com/vllm/vllm-omni/builds/13677/canvas?sid=01a00773-f944-4821-b7df-7465ce7dce30&tab=output
```
FAILED tests/dfx/reliability/invalid_param_test/test_invalid_omni_chat.py::test_chat_completions_invalid_requests[qwen3_omni-invalid_response_format_json_schema] - AssertionError: Expected error text to contain all of ('response_format', 'value_error', 'json_schema'); missing ['value_error']. haystack='{"error": {"message": "When response_format type is \'json_schema\', the \'json_schema\' field must be provided. (parameter=response_format)", "type": "BadRequestError", "param": "response_format", "code": 400}}\n{"error":{"message":"When response_format type is \'json_schema\', the \'json_schema\' field must be provided. (parameter=response_format)","type":"BadRequestError","param":"response_format","code":400}}'
```

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://vllm-omni.readthedocs.io), which can answer lots of frequently asked questions.

--- a/tests/dfx/reliability/invalid_param_test/test_invalid_omni_chat.py
+++ b/tests/dfx/reliability/invalid_param_test/test_invalid_omni_chat.py
@@ -119,7 +119,7 @@ def _chat_completions_request_without_expectations(omni_server: OmniServer, case
         pytest.param(
             "response_format_json_schema_incomplete",
             400,
-            ("response_format", "value_error", "json_schema"),
+            ("response_format", "BadRequestError", "json_schema"),
             id="invalid_response_format_json_schema",
         ),
         pytest.param("logprobs_wrong_type", 400, "logprobs", id="logprobs_wrong_type", marks=_SKIP_ISSUE_3649),

